tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Redis/RedisTags.cs

```diff
@@ -25,5 +25,8 @@ internal partial class RedisTags : InstrumentationTags
 
         [Tag(Trace.Tags.OutPort)]
         public string Port { get; set; }
+
+        [Tag(Trace.Tags.RedisDatabaseIndex)]
```

Comment:
I don't think what I'm going to say makes sense but I believe it is expected to be a metric. Maybe ask Framework and integrations team about it

Reply:
It is supposed to have type int (as opposed to a string or bool) so probably a metric?

Reply:
As discussed offline, yes, it is supposed to be a metric. I've updated the PR to account for this.

For prosperity: always use metrics for "number like" tags. Even though it's not really a "metric" that should be summed/averaged, it's important to record it as such so that we don't use "string-like" searching. e.g. searching for `db.redis.database_index=1` should _only_ bring back `1`, not `10` or `11` as it would otherwise